pkgs/development/libraries/science/math/faiss/default.nix

```diff
@@ -114,32 +94,10 @@ stdenv.mkDerivation {
      python -m pip wheel --verbose --no-index --no-deps --no-clean --no-build-isolation --wheel-dir dist .)
   '';
 
-  postInstall =
-    ''
-      mkdir -p $demos/bin
-      if [[ "$buildInputs" == *demo_ivfpq_indexing* ]] ; then
-        cp ./demos/demo_ivfpq_indexing $demos/bin/
-      fi
-    ''
-    + lib.optionalString pythonSupport ''
-      mkdir "$dist"
-      cp faiss/python/dist/*.whl "$dist/"
-    '';
-
-  passthru = {
-    inherit cudaSupport cudaPackages pythonSupport;
-
-    tests = {
-      runDemos =
```

Comment:
Would be nice to have more thorough integration testing in the long term, but I guess right now this is more of a maintenance burden

Reply:
Agreed. The demos don't seem to go through their CI, alas.